data/SinghRamesh_2023/metadata.yml

```diff
@@ -0,0 +1,459 @@
+      C
+  - find: Elevated
+    value: elevated
+    description: Plants grown at elevated temperatures (5 C above ambient), average/day/night
+      of 28/31/25 C
+  - find: Cold
+    value: cold
+    description: Plants grown at cold temperatures (7 C below ambient), average/day/night
+      of 16.5/19/14 C
+- context_property: seed provenance
+  category: treatment
+  var_in: Elevation_of_origin
+  values:
+  - value: 300
+    description: lowland species growing at 300m elevation
+  - value: 620
+    description: lowland species growing at 620m elevation
+  - value: 990
+    description: upland species growing at 990m elevation
+  - value: 1550
+    description: upland species growing at 1550m elevation
+traits:
+- var_in: LMF
+  unit_in: g/g
+  trait_name: leaf_mass_fraction
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: Final biomass was measured from destructive harvests of three plants per
+    species by treatment combination, except for F. bourjotiana in the nutrient-rich,
+    elevated temperature treatment, where only two plants survived. Thus, a total
+    of 71 plants were harvested for final biomass. The dry mass of leaves, stems and
+    roots were measured separately and used to calculate Above Ground Biomass (AGB,
+    sum of leaf and stem dry mass) and Below Ground Biomass (BGB, total root dry mass).
+    The AGB and BGB were used to calculate the root-to-shoot ratio (R:S).
+- var_in: SMF
+  unit_in: g/g
+  trait_name: stem_mass_fraction
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: Final biomass was measured from destructive harvests of three plants per
+    species by treatment combination, except for F. bourjotiana in the nutrient-rich,
+    elevated temperature treatment, where only two plants survived. Thus, a total
+    of 71 plants were harvested for final biomass. The dry mass of leaves, stems and
+    roots were measured separately and used to calculate Above Ground Biomass (AGB,
+    sum of leaf and stem dry mass) and Below Ground Biomass (BGB, total root dry mass).
+    The AGB and BGB were used to calculate the root-to-shoot ratio (R:S).
+- var_in: RMF
+  unit_in: g/g
+  trait_name: root_mass_fraction
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: Final biomass was measured from destructive harvests of three plants per
+    species by treatment combination, except for F. bourjotiana in the nutrient-rich,
+    elevated temperature treatment, where only two plants survived. Thus, a total
+    of 71 plants were harvested for final biomass. The dry mass of leaves, stems and
+    roots were measured separately and used to calculate Above Ground Biomass (AGB,
+    sum of leaf and stem dry mass) and Below Ground Biomass (BGB, total root dry mass).
+    The AGB and BGB were used to calculate the root-to-shoot ratio (R:S).
```

Comment:
I noticed the column R/S (root_shoot_ratio) isn't mapped - was this intentional?

Reply:
no - I've added it now